packages/theme/src/cli/utilities/theme-environment/theme-environment.ts

```diff
@@ -50,17 +49,6 @@ function ensureThemeEnvironmentSetup(theme: Theme, ctx: DevServerContext) {
   return {
     workPromise: uploadPromise.then((result) => result.workPromise),
     renderProgress: async () => {
-      if (ctx.options.themeEditorSync) {
```

Comment:
This causes the process to hang because`reconcilePromise` will not resolve.

I'm removing this for now, and if we want to add a progress bar back later for the reconciliation step, we can.

Reply:
If that's the case, then shouldn't we change `const uploadPromise = reconcilePromise.then` as well??

Can we make the promise to resolve and keep doing work in the background instead?